tests/unit_tests/test_wave_06.py

```diff
@@ -108,17 +110,15 @@ def test_swap_best_by_category_reordered():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    assert result
```

Comment:
👍  I'd say the same here as above about testing for all three items in each person's inventory.